.github/workflows/cloud-tpu-ci-nightly.yml

```diff
@@ -24,9 +24,12 @@ jobs:
     strategy:
       fail-fast: false # don't cancel all jobs on failure
       matrix:
-        jaxlib-version: ["pypi_latest", "nightly"]
+        jaxlib-version: ["pypi_latest", "nightly", "nightly+oldest_supported_libtpu"]
         tpu-type: ["v3-8", "v4-8", "v5e-4"]
     name: "TPU test (jaxlib=${{ matrix.jaxlib-version }}, ${{ matrix.tpu-type }})"
+    env:
+      DATE_12_WEEKS_AGO: $(python3 -c "import datetime; print(datetime.date.today() - datetime.timedelta(weeks=12))")
```

Comment:
Let's not actually use the rolling window to start and just specify the exact libtpu-nightly version that should be tested, and we can manually update that to start. We'll probably want to automatically roll the version in the future so we don't have to manually update it, but I wanna start on the conservative side (e.g. to make sure our version checks in the code line up with what we're testing)